stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_composer_default_center_content.xml

```diff
@@ -0,0 +1,106 @@
+        android:layout_gravity="center_horizontal"
+        android:layout_marginTop="0dp"
+        app:layout_constraintBottom_toTopOf="@id/messageReplyView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        >
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/attachmentsRecyclerView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="1dp"
+            android:layout_marginTop="@dimen/stream_ui_spacing_medium"
+            android:layout_marginEnd="1dp"
+            android:clipToPadding="false"
+            android:orientation="horizontal"
+            android:paddingHorizontal="@dimen/stream_ui_spacing_tiny"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            />
+
+    </FrameLayout>
+
+    <io.getstream.chat.android.ui.message.list.adapter.view.internal.MessageReplyView
+        android:id="@+id/messageReplyView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        app:layout_constrainedHeight="true"
+        app:layout_constrainedWidth="true"
+        app:layout_constraintBottom_toTopOf="@id/messageEditText"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/selectedAttachmentsContainer"
+        app:streamUiEllipsize="true"
+        />
+
+    <TextView
+        android:id="@+id/commandBadge"
+        android:layout_width="wrap_content"
+        android:layout_height="24dp"
+        android:layout_marginStart="@dimen/stream_ui_spacing_small"
+        android:layout_marginBottom="@dimen/stream_ui_spacing_medium"
+        android:background="@drawable/stream_ui_shape_command_background"
+        android:gravity="center_vertical"
+        android:paddingStart="@dimen/stream_ui_spacing_tiny"
+        android:paddingEnd="@dimen/stream_ui_spacing_small"
+        android:textAllCaps="true"
+        android:textAppearance="@style/StreamUiTextAppearance.FootnoteBold"
+        android:textColor="@color/stream_ui_literal_white"
+        app:drawableStartCompat="@drawable/stream_ui_ic_command_white"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/messageEditText"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:text="giphy"
+        />
+
+    <androidx.appcompat.widget.AppCompatEditText
+        android:id="@+id/messageEditText"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="@null"
+        android:inputType="textCapSentences|textMultiLine"
+        android:maxLines="7"
```

Comment:
Any way to customize all of these hardcoded numbers/attributes?

Reply:
My plan was to use the TransformStyle and style attrs - the same API type that's used in `MessageInputView` and apply those style transformations only to the default contents.